web/app/components/comment/comment.jsx

```diff
@@ -501,9 +527,25 @@ export default class Comment extends Component {
                   <span {...getHandleClickProps(() => this.togglePin(pinned))} className="comment__control">
                     {pinned ? 'Unpin' : 'Pin'}
                   </span>
-                  <span {...getHandleClickProps(() => this.toggleBlock(userBlocked))} className="comment__control">
-                    {userBlocked ? 'Unblock' : 'Block'}
-                  </span>
+
+                  {userBlocked && (
+                    <span {...getHandleClickProps(() => this.onUnblockUserClick())} className="comment__control">
+                      Unblock
+                    </span>
+                  )}
+
+                  {!userBlocked && (
+                    <span className="comment__control comment__control_select-label">
+                      Block
+                      <select className="comment__control_select" onChange={this.onBlockUserClick}>
```

Comment:
Linter asks to use onBlur here instead of onChange.
See https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md

Reply:
ok, add `/* eslint-disable jsx-a11y/no-onchange */` and create a separate issue to further investigate

Reply:
It's done. I've opened #148 issue in regard of this situation.